lib/importers/course_upload_importer.rb

```diff
@@ -10,10 +10,11 @@ def self.update_courses(courses)
     end
   end
 
-  def initialize(course)
-    @course = course
-    @start = course.start
-    @end = course.end + Course::UPDATE_LENGTH
+  def initialize(update_cs)
```

Comment:
I think this and the rest of the importers that currently take a `course` as their main argument should not be changed to take an update service; the update service should be an additional, optional keyword argument.

The idea here is that these importers are mainly concerned about a specific course and its data, and the optional context of the update service is there just for the purpose of logging.